spec/schema.json

```diff
@@ -15,15 +15,97 @@
     }
   ],
   "paths": {
-    "/transactions/{transactionId}/limited-partnership/partnership": {
+    "/transactions/{transaction_id}/incorporation/limited-partnership": {
+      "post": {
+        "tags": [
+          "Limited Partnerships"
+        ],
+        "summary": "Create a Limited Partnership incorporation",
+        "parameters": [
+          {
+            "name": "transaction_id",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Limited Partnership Incorporation has been successfully created.",
+            "schema": {
+              "$ref": "#/definitions/CreatedResponse"
+            }
+          },
+          "400": {
+            "description": "Bad Request"
+          },
+          "401": {
+            "description": "Unauthorised"
+          },
+          "500": {
+            "description": "Internal Server Error"
+          }
+        }
+      }
+    },
+    "/transactions/{transaction_id}/incorporation/limited-partnership/{filing_resource_id}": {
+      "get": {
+        "tags": [
+          "Limited Partnerships"
+        ],
+        "summary": "Get a Limited Partnership incorporation",
+        "parameters": [
+          {
+            "name": "transaction_id",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "filing_resource_id",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "include_sub_resources",
+            "in": "query",
+            "required": true,
+            "type": "boolean",
+            "description": "Whether to include sub-resources or not"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful response",
+            "schema": {
+              "$ref": "#/definitions/LimitedPartnershipIncorporationGetResponse"
+            }
+          },
+          "400": {
```

Comment:
Bad Request 400 isn't mentioned in the Architecture/Design spec. Also for the other GET below.

Reply:
updated